process/constants.go

```diff
@@ -107,6 +107,9 @@ const MinShardHeadersFromSameShardInOneMetaBlock = 10
 // MaxNumOfTxsToSelect defines the maximum number of transactions that should be selected from the cache
 const MaxNumOfTxsToSelect = 30000
 
+// MaxGasBandwidthPerBatchPerSender defines the maximum gas bandwidth that should be selected for a sender per batch from the cache
+const MaxGasBandwidthPerBatchPerSender = 5000000
```

Comment:
What happens if a sender sends a transaction with 510 mil gas? It will take only that transaction?

Reply:
it will include the tx and move to next sender in the iteration